changelogs/changelog.yaml

```diff
@@ -954,3 +954,144 @@ releases:
+        'edit_config' method (https://github.com/ansible-collection/dellemc.enterprise_sonic/pull/443).
+      - sonic_bgp_communities - Fix issues in merged state for standard community-lists
+        (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/440).
+      - sonic_copp - Update reserved CoPP names list (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/481).
+      - sonic_interfaces - Remove the restriction preventing configuration of interface
+        speed for port channel member interfaces (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/470).
+      - sonic_l3_interfaces - Eliminate unconditional sending of the new autoconf
+        REST API option during replaced and overridden state handling (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/474).
+      - sonic_mclag - Delete any remaining PortChannel members for an mclag domain
+        before attempting to delete the mclag domain (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/463).
+      - sonic_ospf_area - Fix OSPF area bug (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/466).
+      - sonic_qos_interfaces - Fix command deletion bug (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/473).
+      - sonic_qos_wred - Update QoS WRED regression test case based on SONiC code
+        changes (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/465).
+      - sonic_stp - Change the criteria for converting vlans and vlan ranges to handle
+        vlan IDs with more than one digit (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/460).
+      - sonic_stp - Fix functionality to allow a value of 0 to be configured for the
+        appropriate integer attributes and refactor module code(https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/477).
+      - sonic_system - Catch the ConnectionError exception caused by unconditional
+        fetching of auditd and ip loadshare hash algorithm configuration, and return
+        empty configuration instead of allowing the uncaught exception to abort all
+        "system" operations on SONiC images older than version 4.4.0 (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/441).
+      - sonic_vrrp - Update delete handling to fix regression failure (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/455).
+      - sonic_vxlan - Fix failing regression tests for sonic_vxlan (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/471).
+      minor_changes:
+      - sonic-logging - Add new 'auditd_system' choice to the 'message_type' choices
+        for the logging resource module (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/459).
+      - sonic-vxlan - Add 'suppress_vlan_neigh' vlan list option (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/448).
```

Comment:
```suggestion
      - sonic_logging - Add new 'auditd_system' choice to the 'message_type' choices
        for the logging resource module (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/459).
      - sonic_vxlan - Add 'suppress_vlan_neigh' vlan list option (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/448).
```

Reply:
Done.

Note: Making this change in the changelog.yaml file caused a re-ordering of the corresponding entries in the CHANGELOG.rst file because the lexical ordering of the module names at the start of the fragments was changed.